data/changelog.txt

```diff
@@ -1,5 +1,70 @@
 # 0.F (2021-04-15)
 
+3974 files changed, 5440251 insertions(+), 3904330 deletions(-)
+10,463 commits
+~486 contributors
+
+New game entities (core): 4510
+Items: 987
+    228 misc items, 199 books, 196 articles of clothing, 161 guns and gun related items,
+    151 comestibles, 99 tools, 53 ammunition types
+Mapgen: 778
+    424 overmap terrains, 198 palettes, 62 start locations, 41 city buildings, 25 map extras,
+    20 overmap specials, 5 overmap locations, 3 region setting
+Crafting: 562
+    223 construction groups, 201 requirements, 60 proficiencies, 50 constructions,
+    17 recipe groups, 10 recipe categories, 1 construction category
+Achievements: 363
+    115 achievements, 107 event statistics, 100 event transformations, 26 score entries, 15 conducts
+ITEM TRAITS: 325
+    171 item groups, 97 materials, 20 ammunition types, 18 ammo effects, 13 tool qualities,
+    2 vitamins, 3 item actions, 1 fault
+Player Traits: 263
+    144 professions, 41 effect types, 28 skills, 17 mutations, 14 activity types,
+    4 skill display types, 3 behaviors, 3 movement modes, 3 scent types, 2 morale types,
+    1 butchery requirement, 1 disease type, 1 mutation types, 1 trait group
+NPCs and NPC interaction 237
+    255 talk topics, 29 missions, 29 scenarios, 15 npcs, 6 npc classes, 3 factions
+Map Traits: 234
+    149 terrains, 12 weather types, 64 furnitures, 5 gates, 3 field types, 1 terrain transformation
```

Comment:
```suggestion
Mapgen (detailed): 234
    149 terrains, 12 weather types, 64 furniture, 5 gates, 3 field types, 1 terrain transformation
```

Reply:
It's not really part of mapgen though, it's definitions of things that appear on the map.

Reply:
```suggestion
Map Components: 234
    149 terrains, 12 weather types, 64 furniture, 5 gates, 3 field types, 1 terrain transformation
```

Reply:
These aren't components, weather types for example aren't a visible thing on the map, and neither are terrain transformations.